docs/local.md

```diff
@@ -29,11 +29,11 @@ Build binary:
 make build
 ```
 
-This creates `swapd` and `swapcli` binaries in the root directory.
+This creates `swapd` and `swapcli` binaries in the `/bin` directory.
```

Comment:
That looks like `/bin` in the root of the filesystem. Maybe say something like: "the `bin` folder at the top of the project"?

Thanks for the changes! Let's make that tiny fix and we'll get it merged.

Reply:
Sure, just made that adjustment